src/web/css/materialdesignicons/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
Do not check in user configuration files => delete this file or add entry to `.gitignore`